File: Iniciante/1015 - Distância Entre Dois Pontos.c
"""
Autor: Pierre Vieira
Data da submissão: 13/09/2018 22:20:57
"""
#include <stdio.h>
#include <math.h>
int main()
{
    float x1, x2, y1, y2, dist;
    scanf("%f %f",&x1, &y1);
    scanf("%f %f",&x2, &y2);
    dist = sqrt(pow(x2 - x1,2)+pow(y2 - y1,2));
    printf("%.4f\n",dist);
}

File: C#/Iniciante/1015 - Distância Entre Dois Pontos.cs
/*
* Autor: Pierre Vieira
* Data de submissão: 13/05/2020 13:53:44
*/
using System;
using System.Globalization;

namespace URI
{
    class URI
    {
        static void Main(string[] args)
        {
            string[] linha1, linha2;
            double x1, x2, y1, y2, distancia;
            linha1 = Console.ReadLine().Split(' ');
            linha2 = Console.ReadLine().Split(' ');
            x1 = double.Parse(linha1[0]);
            y1 = double.Parse(linha1[1]);
            x2 = double.Parse(linha2[0]);
            y2 = double.Parse(linha2[1]);
            distancia = Math.Sqrt((Math.Pow(x2 - x1, 2) + Math.Pow(y2 - y1, 2)));
            Console.WriteLine(distancia.ToString("F4", CultureInfo.InvariantCulture));
        }
    }
}
